src/integration/java/org/humancellatlas/ingest/file/FileControllerTest.java

```diff
@@ -0,0 +1,219 @@
+package org.humancellatlas.ingest.file;
+
+import org.humancellatlas.ingest.config.MigrationConfiguration;
+import org.humancellatlas.ingest.core.MetadataDocument;
+import org.humancellatlas.ingest.core.service.ValidationStateChangeService;
+import org.humancellatlas.ingest.messaging.MessageRouter;
+import org.humancellatlas.ingest.process.Process;
+import org.humancellatlas.ingest.process.ProcessRepository;
+import org.humancellatlas.ingest.state.ValidationState;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc(printOnlyOnFailure = false)
+public class FileControllerTest {
+    @MockBean
+    ValidationStateChangeService validationStateChangeService;
+
+    @Autowired
+    private MockMvc webApp;
+
+    @Autowired
+    private ProcessRepository processRepository;
+
+    @Autowired
+    private FileRepository fileRepository;
+
+    @MockBean
+    private MigrationConfiguration migrationConfiguration;
+
+    @MockBean
+    private MessageRouter messageRouter;
+
+    Process process;
+
+    File file = new File();
+
+    @BeforeEach
+    void setUp() {
+        process = new Process(UUID.randomUUID());
+        processRepository.save(process);
+
+        file = new File();
+        fileRepository.save(file);
+
+    }
+
+    @Test
+    public void testOverrideLinkFileAsInputToMultipleProcessesDefaultPostEndpoint() throws Exception {
+        Process process2 = new Process();
+        processRepository.save(process2);
+
+        webApp.perform(post("/files/{fileId}/inputToProcesses/", file.getId())
+                .contentType("text/uri-list")
+                .content(ServletUriComponentsBuilder.fromCurrentContextPath().build().toUriString() + "/processes/" + process.getId()
```

Comment:
The same comment as in the other test case. Make a variable from this: `ServletUriComponentsBuilder.fromCurrentContextPath().build().toUriString() + "/processes/"`. Tx